elastic/kibana
# [Response Ops] [Connectors] Move connectors type UI components to `stack_connectors` plugin

Towards https://github.com/elastic/kibana/issues/90931

## Summary

Moved connector types UI code from `triggers_actions_ui/public/built_in_action_types` to new `stack_connectors` plugin.

Split connector types into `stack` and `cases`
* stack types are `.email`, `.index`, `.pagerduty`, `.server-log`, `.slack`, `.teams`, `.webhook`
* cases types are `.cases-webhook`, `.jira`, `.resilient`, `.servicenow`, `.servicenow-sir`, `.servicenow-itom`, `.swimlane`, `.xmatters`

Followup PRs to come for splitting out functional tests

## To Verify
Ensure you can access the create connector form for each connector type in all the places in Kibana where you can create connectors. Also ensure that the action params forms work for connectors when adding actions to rules. I looked in the Stack Management > Rules and Connectors UI, the Stack Management > Cases UI, the Uptime Settings UI, the Security rule creation UI
